### ISSUE TITLE
Consider move this to dotnet/docfx repo?

### ISSUE THREAD

**Issue by superyyrrzz** [kns-it/docker-docfx#2]
Hi @kns-it! I am from docfx team. We [plan to release docker image](https://github.com/dotnet/docfx/issues/4385) while releasing new version of docfx.

Given this is the most well-used docfx docker image on docker hub, I think it may be better to use your Docker file instead of starting from scratch by ourselves. I am thinking of integrating this into docfx release pipeline, but I'd like to first hear your thoughts on this.

**Comment by prskr** [comment]
Hi @superyyrrzz feel free to start with a copy of our Dockerfiles!
Right now I'm not sure if the SQLite hack is still necessary and I haven't had time to upgrade to Mono 6 but beside of these two improvements I think the images are fine and I'm using them in multiple GitLab pipelines without any issue.

I could also migrate the Dockerfiles/scripts as a PR if you like?
Haven't thought of this option, yet, because I thought without an official Docker repository the Dockerfile would be useless.

As soon as you are releasing official Docker images I would be glad to archive this repository with a note that there are official images now. I'm also open for other proposals how to handle the transition.

**Comment by superyyrrzz** [comment]
Of course, PR is welcomed! You can create one to dotnet/docfx root folder. I will later integrate it with CI to release it.

I think SQLite is no longer needed as [docfx handles this hack](https://github.com/dotnet/docfx/blob/6beabd7259355103c9ec7c9c5a3c88ae5b7bf5e5/src/docfx/docfx.csproj#L45), but it is ok to keep it there before testing to ensure it work.

I have the same thoughts with you about the transition. I will go back to this issue and let you know once official images are ready. We can sync in this issue 😄
